share/module_soil_pre.F

```diff
@@ -224,7 +224,7 @@ SUBROUTINE adjust_for_seaice_post ( xice , landmask , tsk_old , tsk , ivgtyp , v
 
          CASE ( SLABSCHEME )
 
-         CASE ( LSMSCHEME , NOAHMPSCHEME , CLMSCHEME, SSIBSCHEME )   !mchen add for ssib
+         CASE ( LSMSCHEME , NOAHMPSCHEME , CLMSCHEME, CTSMSCHEME, SSIBSCHEME )   !mchen add for ssib
```

Comment:
@negin513 
Negin,
Likely this CTSMSCHEME (and at line 3219) are causing NMM troubles. This probably needs to be defined in the file Registry/Registry.NMM via a package. You can make sure that NMM never uses it (if you want) with a test in check_a_mundo.

Reply:
@davegill can I use instead? I think this would be simpler and we used WRF_USE_CTSM in other places in the code. 

```Fortran
#ifdef WRF_USE_CTSM 
CASE ( LSMSCHEME , NOAHMPSCHEME , CLMSCHEME, CTSMSCHEME, SSIBSCHEME )   !mchen add for ssib
#else
CASE ( LSMSCHEME , NOAHMPSCHEME , CLMSCHEME, SSIBSCHEME )   !mchen add for ssib
#endif

Reply:
Negin,
You bet. That would be a fine way to do it. 
Also ifdef the location at the top of the surface driver where the CTSMSCHEME is pulled in from module_state_description (if not done already).

Reply:
Dave, 
Thanks for your comment. In the latest push, I made this change (adding #ifdef WRF_USE_CTSM), which resolved the issue with the tests. 

Also, in the surface driver, we have a similar logic to not use CTSMSCHEME for NMM cores. 
https://github.com/wrf-model/WRF/blob/1acbe6a4dd6a0dc2bd10c4b708647de90be4fa39/phys/module_surface_driver.F#L298-L344

Thanks for your help with this.